content/news/release-20150614.md

```diff
@@ -0,0 +1,51 @@
+We are proud to announce the June 2015 release of OpenRA!
+
+As promised, it was our goal to reduce the length of this development cycle, and we succeeded! With about 22500 changes lines in just over 300 commits by a total of 25 developers since last release, the list of new features is naturally a bit shorter than last time around, but it contains some good ones!
+
+The standout feature of this release is of course the long-awaited in-game map editor. For the first time ever, we are offering a map editing ability for all our mods, on all supported operating systems. You can access the map editor from the "Extras" sub-menu of the main menu.
+
+The new map editor offers the following benefits and features:
+
+* Fully cross-platform, just like the game itself, while the legacy editor did not work on OS X.
+* Truely WYSIWYG. It uses the game renderer, so there are no more rendering glitches, missing building roofs or turrets.
+* Support for all mods. The legacy editor did not support Dune 2000 properly, and Tiberian Sun not at all.
+
+Compared to the legacy editor, there is not yet full feature parity: the in-game editor is missing cut-and-paste operations, a display of the sum of resources on the map, and also does not show the coordinates of cells. These short-comings will be rectified in future releases. Meanwhile, they are still available in the legacy editor, which is still being shipped with this release. It will, however, be removed in the not too distant future.
+
+Complementing the new editor is an improved map chooser dialog, which now separates official and custom maps. It also allows deleting one or all of your custom maps. We also took this chance to add the often-requested ability for all players to view the map chooser from the game lobby (but changing the map still remains the prerogative of the host).
+
+Another new feature with a lot of potential is the added operating system integration of OpenRA. The installers will associate the game with OpenRA replay files and OpenRA server URLs. These will allow websites hosting replay archives to add one-click watching of replays, and creating alternative server browsers. As an example, our own [web-based server browser](/games/) has been enhanced to offer clickable links to waiting games that will automatically start OpenRA and connect you to the server. You can also create bookmarks to your favourite OpenRA servers on your desktop! These associations are available on all supported operating systems as well.
```

Comment:
I'd maybe be a bit careful about the "all supported operating systems", since this will somewhat depend on what our various downstream linux packagers do.